Formula/porg.rb

```diff
@@ -0,0 +1,26 @@
+class Porg < Formula
+  desc "Source Code Package Organizer"
+  homepage "http://porg.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/porg/porg-0.10.tar.gz"
+  sha256 "48f8433193f92097824ed7a72c4babafb29dc2ffe60e7df3635664f59f09cedd"
+
+  option "without-grop", "Don't install Grop, which is the graphic interface of porg."
```

Comment:
OK.  I'll change it.

I thought `without-grop` would be easier for the users to understand that `Grop` won't be installed, but in `without-gtkmm3` case the description can still explain that to the users.